analyses/focal-cn-file-preparation/03-add-cytoband-status-consensus.Rmd

```diff
@@ -0,0 +1,160 @@
+---
+title: "Add dominant status column to consensus SEG file with cytoband field"
+output: 
+  html_notebook:
+    toc: TRUE
+    toc_float: TRUE
+author: Chante Bethell for ALSF CCDL
+date: 2020
+---
+
+This notebook adds dominant status information per cytoband to the consensus SEG files prepared in `run-prepare-cn.sh` using the `bedtools coverage` function (these files are stored in the project's scratch directory as bed files).
+
+## Usage
+
+This notebook is intended to be run from the command line with the following (assumes you are in the root directory of the repository):
+
+```
+Rscript -e "rmarkdown::render('analyses/focal-cn-file-preparation/03-.Rmd', clean = TRUE)"
+```
+
+## Set up
+
+### Libraries and functions
+
+```{r}
+library(tidyverse)
+```
+
+### Files and directories
+
+```{r}
+scratch_dir <- file.path("..", "..", "scratch")
+```
+
+### Read in files 
+
+The files we are reading in here are the bed files prepared in the shell script using `bedtools coverage`.
+
+```{r message = FALSE}
+all_callable_cytoband_status_df <-
+  read_tsv(file.path(scratch_dir, "intersect_with_cytoband_callable.bed"),
+                  col_names = FALSE)
+
+loss_cytoband_status_df <-
+  read_tsv(file.path(scratch_dir, "intersect_with_cytoband_loss.bed"),
+                  col_names = FALSE)
+
+gain_cytoband_status_df <-
+  read_tsv(file.path(scratch_dir, "intersect_with_cytoband_gain.bed"),
+                  col_names = FALSE)
+```
+
+## Wrangle and merge consensus cytoband status data
+
+### Filter each of the cytoband status data.frames
+
+```{r}
+all_callable_cytoband_status_df <-
+  all_callable_cytoband_status_df %>%
+  select(
+    chr = X1,
+    cytoband = X4,
```

Comment:
I am adding in `band_length` here so that we can output this to properly calculate the fractions of each arm that are gains, losses, etc. in the next step. You won't need to keep it for any of the other data frames, as it will be the same, so the join will retain the info. 
```suggestion
    cytoband = X4,
    band_length = X8,
```